staker/block_validator.go

```diff
@@ -84,10 +92,9 @@ type BlockValidator struct {
 
 type BlockValidatorConfig struct {
 	Enable                      bool                          `koanf:"enable"`
-	ValidationServer            rpcclient.ClientConfig        `koanf:"validation-server" reload:"hot"`
```

Comment:
If we're getting rid of this option we might want to put this in v2.3.4 after all, so we don't have two back to back releases both making breaking changes to this config. Or maybe we could keep this option around but not have it do anything, and log a warning saying it's deprecated? I think I'd prefer the former.

Reply:
Renamed execution back to Validation so we're in-line with 2.3.3 API.. can be released as 2.3.4-rc5 if you prefer